lib/geocoder/lookups/esri.rb

```diff
@@ -15,6 +16,24 @@ def query_url(query)
         url_query_string(query)
     end
 
+    def generate_token(expires=1440)
```

Comment:
Putting this method into the EsriToken class would be cleaner, could you move it over? Thanks